make/test/JtregNativeJdk.gmk

```diff
@@ -110,6 +110,20 @@ ifeq ($(call isTargetOs, linux), true)
   BUILD_JDK_JTREG_LIBRARIES_LDFLAGS_libCreationTimeHelper := -ldl
 endif
 
+# Kerberos native test library - only build if krb5 is available
+ifeq ($(ENABLE_LIBLINUXKRB5), true)
+  BUILD_JDK_JTREG_LIBRARIES_LDFLAGS_libNativeCredentialCacheHelper := $(KRB5_LIBS)
+  BUILD_JDK_JTREG_LIBRARIES_CFLAGS_libNativeCredentialCacheHelper := $(KRB5_CFLAGS)
+
+  ifeq ($(call isTargetOs, macosx), true)
+    # On macOS, disable deprecation warnings for krb5 API
+    BUILD_JDK_JTREG_LIBRARIES_CFLAGS_libNativeCredentialCacheHelper += -Wno-deprecated-declarations
+  endif
```

Comment:
Isn't this enabled and should be tested on macos regardless of `ENABLE_LIBLINUXKRB5` as support was there already?